src/manifold/stream/queue.clj

```diff
@@ -2,7 +2,6 @@
   {:no-doc true}
   (:require
     [clj-commons.primitive-math :as p]
-    [manifold.stream.graph :as g]
```

Comment:
Hmmm, this is a little suspicious. 

It looks like it can be safely removed, but it used to be the case that some namespaces were required to force initialization of necessary data structures that were indirectly needed. Yet another reason indirect requires are a bad idea.

Removing this looks ok afaict, tho.